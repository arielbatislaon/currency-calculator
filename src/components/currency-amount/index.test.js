import React from 'react';
import renderer from 'react-test-renderer';

import CurrencyAmount from './index';

it('renders CurrencyAmount correctly ', () => {
  const tree = renderer.create(<CurrencyAmount />).toJSON();
  expect(tree).toMatchSnapshot();
});