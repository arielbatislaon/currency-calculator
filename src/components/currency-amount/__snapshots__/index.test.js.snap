// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CurrencyAmount correctly  1`] = `
<label
  className=""
  id="amountLabel"
>
  <div
    aria-labelledby="amountLabel"
    className="label"
  >
    Amount to convert:
  </div>
  <input
    aria-labelledby="amountLabel"
    onChange={null}
    placeholder="Enter Amount"
    type="text"
    value={1}
  />
  
</label>
`;
